lib/easypost/credit_card.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Credit card object that user can fund or delete
+class EasyPost::CreditCard < EasyPost::Resource
+  # Fund your EasyPost wallet by charging your primary or secondary card on file.
+  def self.fund(amount, primary_or_secondary = 'primary', api_key = nil)
+    payment_methods = EasyPost::PaymentMethod.all
+    payment_method_map = {
+      'primary' => 'primary_payment_method',
+      'secondary' => 'secondary_payment_method',
+    }
+
+    payment_method_to_use = payment_method_map[primary_or_secondary]
```

Comment:
If someone passed `tertiary` in as the payment method, this lookup would fail with a KeyError. You'll want to set it to nil if it can't be found (try testing for yourself).

If you set it to nil if it can't find anything in the `payment_method_map`, then you can keep the rest of your code as is and it'll catch it.

Reply:
If the key does not exist, the value will be `nil`, according to this article: https://riptutorial.com/ruby/example/1039/setting-default-values#:~:text=By%20default%2C%20attempting%20to%20lookup,not%20exist%20will%20return%20nil%20.

I also tested it in a compiler, the value will be `nil` if the user enters key other than `primary` and `secondary`